engine/examples/running_ssh_service.md

```diff
@@ -38,6 +38,16 @@ EXPOSE 22
 CMD ["/usr/sbin/sshd", "-D"]
 ```
 
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
```

Comment:
Merge conflicts here

Reply:
Thanks, this has been resolved.

Reply:
I've addressed in a separate commit. @thaJeztah , I'm hesitant to leave "use without-password instead of prohibit-password for Ubuntu 14.04" since 14.04 has reached EOL.